openshift/core.config.yaml

```diff
@@ -36,3 +36,5 @@ objects:
     openshift.tenant.masterurl: https://openshift.local/
     auth_notapproved_redirect: ""
     feature.workitem.remote: "false"
+    deployments.serviceurl: "http://wit:8080"
```

Comment:
https://console.rh-idev.openshift.com/console/project/dsaas-preview/browse/services/core?tab=details
just `http://core`  should do right?

Reply:
done